x/bank/keeper/keeper.go

```diff
@@ -63,8 +63,17 @@ func (k BaseKeeper) GetTotalSupply(ctx sdk.Context) sdk.Coins {
 	return balances.Sort()
 }
 
+// NewBaseKeeper returns a new BaseKeeper object with a given codec, dedicated
+// store key, an AccountKeeper implementation and a parameter Subspace used to
+// store and fetch module parameters. In addition, the BaseKeeper accepts a
+// black-list map. This list describes the set of addresses which are not allowed
+// to receive funds through direct and explicit actions, e.g. via a MsgSend or
+// via a SendCoinsFromModuleToAccount execution.
```

Comment:
```suggestion
// NewBaseKeeper returns a new BaseKeeper object with a given codec, dedicated
// store key, an AccountKeeper implementation, and a parameter Subspace used to
// store and fetch module parameters. The BaseKeeper also accepts a
// blocklist map. This blocklist describes the set of addresses that are not allowed
// to receive funds through direct and explicit actions, for example, by using a MsgSend or
// by using a SendCoinsFromModuleToAccount execution.
```

Reply:
@alexanderbez thank you for this documentation! 
We want to start using inclusive language (blocklist instead of blacklist), see [Google developer documentation style guide](https://developers.google.com/style/word-list#letter-b).
We can also avoid Latin. Instead of "via" we can use "by using"

Reply:
While I totally agree with you, this is how the nomenclature exists in the code. I recommend we keep these changes and change the nomenclature along with any references in the docs in a subsequent PR.

Reply:
Alternatively, we can include your changes and just have a divergence in nomenclature in code vs docs. WDYT @alessio ?

Reply:
Wait, i'm totally wrong! We already made the switch. ACK 🎉

Reply:
I recommend that whenever we see a non-inclusive term, we change it (code and docs). Congratulations for changing this term already in the code @alexanderbez !